ckanext/data_qld/plugin.py

```diff
@@ -304,6 +314,339 @@ def get_blueprint(self):
     def get_commands(self):
         return click_cli.get_commands()
 
+    # IFacets
+    def dataset_facets(self, facets_dict, package_type):
+        new_facets_dict = OrderedDict()
+        new_facets_dict['dataset_type'] = tk._('Data Portals')
+
+        return OrderedDict(list(new_facets_dict.items()) + list(facets_dict.items()))
+
+    # CKANHarvester
+    config = None
```

Comment:
just wondering if this could be done via another class instead of in the main plugin?

Reply:
It would be possible, and I considered it, but I didn't want to have an extra plugin definition in the config.